### PR TITLE
Added Forseti Visualizer section on the website

### PR DESCRIPTION
**Pull request by red2k18**
Added Forseti Visualizer section on the website in an effort to improve website documentation.

### CODE REVIEW

File: _docs/_latest/configure/forseti-visualizer/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Forseti Visualizer
```
Comment (blueandgold) [inline _docs/_latest/configure/forseti-visualizer/index.md]: See comment below on not having this page.

File: _docs/_latest/configure/index.md [hunk 2]
```diff
@@ -44,6 +44,12 @@ Google Cloud Platform (GCP) resources.
 Configure Cloud Profiler to view and analyze CPU usage and memory-allocation of your Forseti application on a 
 Google Cloud Platform (GCP) interface.
 
+**[Configuring Forseti Visualizer]({% link _docs/latest/configure/forseti-visualizer/index.md %})**
```
Comment (blueandgold) [inline _docs/_latest/configure/index.md]: Link to the github page from here.  This way, we don't have a separate page like above that we have to maintain and keep in sync with the github description.  Less page is also more performant.

File: _docs/_latest/configure/forseti-visualizer/index.md (hunk 1 above)
Comment (red2k18) [reply _docs/_latest/configure/forseti-visualizer/index.md]: Done

File: _docs/_latest/configure/index.md (hunk 2 above)
Comment (red2k18) [reply _docs/_latest/configure/index.md]: Done

**Pull request merged**